### PR TITLE
feat(Migration developer): Starting the migration of the developer site

### PR DESCRIPTION
**Pull request by nbaenam**
Migrate the Build on New Relic UI docs (minus the NR1 SDK component library docs) to the Build on New Relic UI docs site.

This is the ticket: NR-258525

### CODE REVIEW

File: src/nav/new-relic-solutions.yml [hunk 1]
```diff
@@ -149,7 +149,21 @@ pages:
   - title: Build on New Relic UI
     pages:
       - title: New Relic CLI
-        path: /docs/new-relic-solutions/tutorials/new-relic-cli
+        path: /docs/new-relic-solutions/tutorials/new-relic-cli
```
Comment (clarkmcadoo) [inline src/nav/new-relic-solutions.yml:152]: I don't think a nav item can have both pages and a path

File: src/nav/new-relic-solutions.yml [hunk 2]
```diff
@@ -149,7 +149,21 @@ pages:
...
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-nerdpack
+          - title: Subscription commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-subscription
+          - title: Plugin commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-plugins
+          - title: Catalog commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-catalog
       - title: Build your own New Relic apps
         path: /docs/new-relic-solutions/new-relic-one/build-custom-new-relic-one-application
       - title: Create a "Hello, World!" app in New Relic
```
Comment (clarkmcadoo) [inline src/nav/new-relic-solutions.yml:169]: The quotes in this line were causing the issue

Comment (nbaenam) [reply src/nav/new-relic-solutions.yml:169]: Thank you so much, @clarkmcadoo. It's working now!

**Review by clarkmcadoo — APPROVED** [review]

File: src/nav/new-relic-solutions.yml [hunk 3]
```diff
@@ -149,7 +149,37 @@ pages:
   - title: Build on New Relic UI
     pages:
       - title: New Relic CLI
-        path: /docs/new-relic-solutions/tutorials/new-relic-cli
+        path: /docs/new-relic-solutions/tutorials/new-relic-cli    
+      - title: New Relic One CLI commands
```
Comment (paperclypse) [inline src/nav/new-relic-solutions.yml:153]:
```suggestion
      - title: New Relic CLI commands
```

File: src/nav/new-relic-solutions.yml [hunk 4]
```diff
@@ -149,7 +149,37 @@ pages:
...
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-config
+          - title: Nerdpack commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-nerdpack
+          - title: Subscription commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-subscription
+          - title: Plugin commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-plugins
+          - title: Catalog commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-catalog
+      - title: New Relic CLI
```
Comment (paperclypse) [inline src/nav/new-relic-solutions.yml:167]:
```suggestion
      - title: CLI reference
```

File: src/nav/new-relic-solutions.yml (hunk 3 above)
Comment (paperclypse) [reply src/nav/new-relic-solutions.yml:153]: @nbaenam "New Relic One" isn't really a name we use anymore. Let's drop the "One" where we see it

File: src/nav/new-relic-solutions.yml (hunk 4 above)
Comment (paperclypse) [reply src/nav/new-relic-solutions.yml:167]: Move this to the top of the "CLI commands" category

File: src/nav/new-relic-solutions.yml [hunk 5]
```diff
@@ -149,7 +149,37 @@ pages:
...
+        path: /docs/new-relic-solutions/build-nr-ui/nerdpack-file-structure
+      - title: Intro to NerdStorage
+        path: /docs/new-relic-solutions/build-nr-ui/nerdstorage
+      - title: Intro to NerdStorageVault
+        path: /docs/new-relic-solutions/build-nr-ui/nerdstoragevault
+      - title: Query and store data
+        path: /docs/new-relic-solutions/build-nr-ui/query-and-store-data
+      - title: Custom visualizations
+        pages:
+          - title: Configuration options
```
Comment (paperclypse) [inline src/nav/new-relic-solutions.yml:179]: The nav title needs to match the page title

**Review by paperclypse — CHANGES_REQUESTED** [review]
Looks good. There are a few things to fix, but otherwise it's pretty close

**Review by paperclypse — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**